tests/functional/spec/question_definitions.spec.js

```diff
@@ -61,5 +61,21 @@ describe('Component: Definition', function() {
 
     });
 
+
+    it('When I click the second definition\'s title link then the description and "Hide this" button for the second definition should be visible', function() {
+
+      return browser
+        .isVisible(DefinitionPage.definitionContent2()).should.eventually.be.false
+        .isVisible(DefinitionPage.definitionButton2()).should.eventually.be.false
+
+        // When
+        .click(DefinitionPage.definitionTitle2())
+
+        // Then
+        .waitForVisible(DefinitionPage.definitionContent2(), 300)
+        .waitForVisible(DefinitionPage.definitionButton2(), 300);
```

Comment:
Do these assert? As fas as I can tell they don't fail if the elements are not visible?

Reply:
Yes, they should fail if the element doesn't become visible within 300ms.